### PR TITLE
Failing test using ienumerable for #1874

### PR DESCRIPTION
**Pull request by danspam**
Failing test for #1874, as requested

### CODE REVIEW

File: src/NHibernate.Test/Extralazy/ExtraLazyFixture.cs [hunk 1]
```diff
@@ -106,6 +107,51 @@ public void OrphanDelete()
...
+				s.Persist(gavin);
+				t.Commit();
+			}
+
+			using (var s = OpenSession())
+			using (var t = s.BeginTransaction())
+			{
+				gavin = s.Get<User>("gavin");
+				widget = s.Get<Widget>("widg1");
+				//Assert.IsTrue(gavin.Widgets.Any(w => w == widget));
```
Comment (hazzik) [inline src/NHibernate.Test/Extralazy/ExtraLazyFixture.cs]:
```suggestion
```

Comment (danspam) [reply src/NHibernate.Test/Extralazy/ExtraLazyFixture.cs]: Un-commenting this line makes the test pass, demonstrating the issue.

Comment (hazzik) [reply src/NHibernate.Test/Extralazy/ExtraLazyFixture.cs]: I understand, could you then put this into the comment?

**Comment by hazzik** [comment]
thanks

**Comment by hazzik** [comment]
https://hibernate.atlassian.net/browse/HHH-5742

**Comment by hazzik** [comment]
Replaced by #1878

**Pull request closed without merging**